### PR TITLE
chore: [sveltekit-pod] 00 Initialize showcase with KIT

### LINKED ISSUE #667: [sveltekit-pod] 00 Initialize showcase with KIT
# Initialize showcase with KIT

## Background

As a developer, I would like my starting base to be the existing KIT for the current technology. It is expected for a KIT to be fully ready before commencing work on the showcase.

## Acceptance

- [ ] Import the KIT
- [ ] Follow the instruction in the Read me to modify required data (ex title, name, description)
- [ ] If you find anything missing or wrong in the KIT, raise a Ticket (even if it is just a high risk update in the package.json)

### PR DESCRIPTION
**Pull request by ihardz**
resolves #667

> # Initialize showcase with KIT
> 
> ## Background
> 
> As a developer, I would like my starting base to be the existing KIT for the current technology. It is expected for a KIT to be fully ready before commencing work on the showcase.
> 
> ## Acceptance
> 
> - [x] Import the KIT
> - [x] Follow the instruction in the Read me to modify required data (ex title, name, description)
> - [x] If you find anything missing or wrong in the KIT, raise a Ticket (even if it is just a high risk update in the package.json)
>

### CODE REVIEW

File: svelte-kit-scss/tests/test.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { expect, test } from '@playwright/test';
+
+test('about page has expected h1', async ({ page }) => {
+  await page.goto('/about');
+  expect(await page.textContent('h1')).toBe('About this app');
+});
```
Comment (flakolefluk) [inline svelte-kit-scss/tests/test.ts:6]: Probably a starter kit thing to fix, I don't think this test is passing.

File: svelte-kit-scss/README.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+## Commands
+
+- `yarn dev` - Starts the development server.
+- `yarn test` - Runs the unit tests.
+- `yarn test:e2e` - Runs the playwright e2e tests.
+- `yarn storybook` - Starts the Storybook UI.
+- `yarn format` - Formats code with prettier for the entire project.
+- `yarn build` - Builds the project for production.
+- `yarn start` - Starts the dev server new tab.
```
Comment (flakolefluk) [inline svelte-kit-scss/README.md:104]: probably a starter kit thing too, but why are we suggesting yarn by default? sveltekit suggests npm and pnpm

**Review by flakolefluk — COMMENTED** [review]
looks good to me.
just raising a few comments/questions

**Review by flakolefluk — APPROVED** [review]
great :) 

:shipit:

File: svelte-kit-scss/README.md (hunk 2 above)
Comment (dustinsgoodman) [reply svelte-kit-scss/README.md]: This is just the default in the main template fwiw.

**Pull request merged**